### ISSUE TITLE
Empty value with trailing comma: comma is not preserved

### ISSUE THREAD

**Issue by aidanheerdegen** [marshallward/f90nml#18]
f90nml turns this:
```fortran
# Input datasets
 &PARM05
 bathyFile='topog.box',
 hydrogThetaFile=,
 hydrogSaltFile=,
 zonalWindFile='windx.sin_y',
 meridWindFile=,
 &
```
into this:
```fortran
&parm05
    bathyfile = 'topog.box'
    hydrogthetafile =
    hydrogsaltfile =
    zonalwindfile = 'windx.sin_y'
    meridwindfile =
/
```
and MITgcm doesn't like those empty entries. I'll leave it to you and MITgcm to figure out who is right.

**Comment by marshallward** [comment]
Do you mean that commas are needed?

You can enable these with `nml.enable_commas = True`

**Comment by aidanheerdegen** [comment]
Oh ok. I'll file an issue with payu   :)

**Comment by marshallward** [comment]
I blame the namelist file (if they're vectors then put the : in the
variable name) but maybe commas should be default on.

**Comment by marshallward** [comment]
This is fixed in v0.15. Closing

**Comment by marshallward** [comment]
https://www.youtube.com/watch?v=D-Vzf5E8UkU
